tests/FSharpx.Collections.Tests/ByteStringTest.fs

```diff
@@ -13,6 +13,10 @@ module ByteStringTests =
           "empty1",   ByteString.create ""B,   ByteString.create ""B,    0
           "empty2",   ByteString.create ""B,   ByteString.create "a"B,  -1  
           "empty3",   ByteString.create "a"B,  ByteString.create ""B,    1
+          "empty4",   ByteString.empty,        ByteString.empty,         0
+          "empty5",   ByteString.empty,        ByteString.create "a"B,  -1  
+          "empty6",   ByteString.create "a"B,  ByteString.empty,         1
+          "empty7",   ByteString.empty,        ByteString.create ""B,    0
```

Comment:
This results in `NullReferenceException`:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at FSharpx.Collections.ByteString.Compare(ByteString a, ByteString b) in C:\Users\grzeg\repos\FSharpx.Collections\src\FSharpx.Collections\ByteString.fs:line 35
```

Reply:
https://github.com/gdziadkiewicz/FSharpx.Collections/actions/runs/3043148096/jobs/4902097301#step:7:491